VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -196,7 +193,9 @@
                         android:layout_height="@dimen/course_unit_nav_bar_btn_height"
                         android:background="@color/white"
                         android:text="@string/assessment_previous"
-                        android:textColor="?attr/buttonTextColor" />
+                        android:textColor="?attr/buttonTextColor"
+                        android:gravity="center_vertical|start"
```

Comment:
Hmm, so it looks like `Button` has a `minWidth` defined, so the `wrap_content` layout width includes that as well. We could explicitly set the `minWidth` to `0dp`, but I think we might as well convert this view to a `TextView` or `ETextView` as appropriate. This would enable us to remove the `background` override as well.